src/closing-disclosure/closing-disclosure-1.html

```diff
@@ -35,7 +35,7 @@
     },
     {
         "term": "Check your interest rate",
-        "definition": "<p>If it doesn't reflect what you were expecting, ask your lender why. If you locked your rate, your lender is only allowed to change it under limited circumstances.</p><p><a href=\"http://www.consumerfinance.gov/askcfpb/143/whats-a-lock-in-or-a-rate-lock.html\">Learn more</a></p>",
+        "definition": "<p>If it doesn't reflect what you were expecting, ask your lender why. If you locked your rate, your lender is only allowed to change it under limited circumstances.</p><p><a href=\"http://www.consumerfinance.gov/askcfpb/143/whats-a-lock-in-or-a-rate-lock.html\" target="_blank">Learn more about rate locks</a></p>",
```

Comment:
It looks like the 500 error on these pages is happening because the quotation marks in `target="blank"` need to be escaped: `target=\"blank\"`. 

If you run `sheer serve --debug` instead of just sheer serve, it'll show any jinja template errors in the page instead of that unhelpful 500 error message. :)